lib/models/file.dart

```diff
@@ -65,7 +65,8 @@ class File {
     file.localID = asset.id;
     file.title = asset.title;
     file.deviceFolder = pathName;
-    file.location = Location(asset.latitude, asset.longitude);
+    var latLng = await asset.latlngAsync();
+    file.location = Location(latLng.longitude, latLng.latitude);
```

Comment:
This impacts first time load performance significantly. Which is why we do this in `FileUploader` instead.

Reply:
Oh. Got it. Will close this.